omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js

```diff
@@ -256,9 +262,12 @@ function PatientSearchWidget(configuration){
                 }
                 var identifier = patient.patientIdentifier != null ? patient.patientIdentifier.identifier : null;
                 if(_.contains(initialPatientUuids, patient.uuid)){
-                    identifier = patient.patientIdentifier.identifier+
```

Comment:
Before i merge, did you intentionally drop this? identifier = patient.patientIdentifier.identifier